doc/adr/0002-repository-standardization.rst

```diff
@@ -0,0 +1,118 @@
+ADR 2: Repository Standardization
+=======================================
+
+Date: 2022-07-06
+
+Authors
+---------
+
+Lorenzo Calegari <lorenzo.calegari@iohk.io>
+
+Status
+------
+
+Draft
+
+Context
+-------
+
+IOG is undertaking a company-wide effort to restructure and standardize its 
+repositories, favoring mono-repos and enforcing shared GitOps and DevOps 
+processes. Parallel to this, a new CI infrastructure is being developed.
+
+Examples of this are:
+
+* `input-output-hk/cardano-world <https://github.com/input-output-hk/cardano-world>`_
+* `input-output-hk/ci-world <https://github.com/input-output-hk/ci-world>`_
+* `input-output-hk/atala-world <https://github.com/input-output-hk/atala-world>`_
+  
+This initiative appears to be championed by the SRE team who are the creators of 
+`divnix/std <https://github.com/divnix/std>`_. Indeed `std` is at the heart of 
+the standardization dream.
+
+Decision
+--------
+
+* Standardization of the repositories has been deemed a worthwhile endeavour, 
+  though of very low priority. 
+* Phase 1 of the standardization process will be carried out in parallel with 
+  :ref:`Move Marconi to a separate repository <separate_marconi_repository>`.
+  A separate repository will be created for Marconi, and from the very beginning 
+  it will use `std`. This way the benefits, limitations and integration costs of 
+  `std` can be experienced and measured, and an informed, definitive decision on 
+  standardizing `plutus-core` and `plutus-apps` themselves can be made. 
```

Comment:
I also think we could just try out std-izing `plutus`, since it only really has binary packaging, so we could layer it over the top fairly quickly I think.